### ISSUE TITLE
Feature: If `[type] | null | undefined`, sort first [type]

### ISSUE THREAD

**Issue by askew07** [azat-io/eslint-plugin-perfectionist#61]
### What rule do you want to change?

sort-union-types

### Describe the problem

@azat-io 

Currently, if `sort-union-types` finds `[type] | null | undefined` it sorts like `null | [type] | undefined`. It sorts alphabetically by default, but has the side-effect of not placing the `[type]` at the begining, when a type might be `null` | `undefined`.

### Code example

this:

```ts
Promise<string | null | undefined> // which reads better
```

becomes

```ts
Promise<null | string | undefined>
```
---------------

and this:
```ts
personId?: string | null // reads much better with the `[type]` sorted at the begining
```
becomes
```ts
personId?: null | string
```

### Additional comments

When finding `[type]` with `null` and/or `undefined`, it reads way better by sorting the `[type]` at the begining of the union.

### Validations

- [X] Read the [docs](https://eslint-plugin-perfectionist.azat.io).
- [X] Check that there isn't [already an issue](https://github.com/azat-io/eslint-plugin-perfectionist/issues) that reports the same bug to avoid creating a duplicate.

**Comment by azat-io** [comment]
Perhaps we should add a special option that would allow nullable types to be placed at the end of the list. Good idea!

**Comment by azat-io** [comment]
Added in 1e391ddf07cf47e0dfa0738ede56335c283fc2ec.

Release is coming soon.

**Comment by azat-io** [comment]
Released in [v2.0.0](https://github.com/azat-io/eslint-plugin-perfectionist/releases/tag/v2.0.0)

**Comment by szamanr** [comment]
for anyone finding this in the future: this is now done using groups. the config below puts null and undefined at the end.

```
    "perfectionist/sort-union-types": ["error", {
      "groups": [
        "unknown",
        "nullish"
      ]
    }]
```
